logging/logforwarding/rsyslog/insecure/rsyslogserver_configmap.yaml

```diff
@@ -0,0 +1,21 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: rsyslogserver
+  labels:
+    provider: aosqe
+    component: "rsyslogserver"
+data:
+  rsyslog.conf: |+
+    global(processInternalMessages="on")
+    module(load="imptcp")
+    module(load="imudp" TimeRequery="500")
+
+    input(type="imptcp" port="514")
+    input(type="imudp" port="514")
+    
+    :programname, contains, "kubernetes.var.log.containers" {
+      if $msg contains "namespace_name=openshift" or $msg contains "namespace_name=default" or $msg contains "namespace_name=kube" then /var/log/custom/infra-container.log
+      if not ($msg contains "namespace_name=openshift" or $msg contains "namespace_name=default" or $msg contains "namespace_name=kube") then /var/log/custom/app-container.log
+    }
+    :programname, !contains, "kubernetes.var.log.containers" /var/log/custom/infra.log
```

Comment:
Still need a method to save systemd logs https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-25365